src/azure-cli/azure/cli/command_modules/synapse/manual/operations/accesscontrol.py

```diff
@@ -6,7 +6,7 @@
 from knack.util import CLIError
 from azure.cli.core.azclierror import InvalidArgumentValueError, ArgumentUsageError
 from azure.cli.core.util import is_guid
-from azure.graphrbac.models import GraphErrorException
+from azure.cli.command_modules.role._msgrpah import GraphError
```

Comment:
`_msgrpah` is protected, so please don't import from it. 

Please see https://github.com/Azure/azure-cli/blob/dev/doc/microsoft_graph_client.md#error

Reply:
so I just need to remove this import statement and GraphError still is available?

Reply:
@jiasli please help to review my new commit. Thanks.

Reply:
> so I just need to remove this import statement and GraphError still is available?

Yes.